lib/iiif_print.rb

```diff
@@ -155,6 +155,10 @@ def self.default_fields(fields: config.metadata_fields)
   end
 
   def self.fields_for_allinson_flex(fields: allinson_flex_fields)
+    sort_order = IiifPrint.config.iiif_metadata_sorter_fields
+    fields = sort_af_fields!(fields, sort_order) if sort_order
+    # removes duplicate named fields
+    fields.uniq!(&:name)
```

Comment:
Why are we getting duplicates?  (The comment says what the code does but not why)

Reply:
I outlined some details in [this](https://github.com/scientist-softserv/essi/issues/20#issuecomment-1528099128) comment, in short, i think it's because of the how the m3 profile gets set up.  I was thinking about making this part configurable but ultimately put this implementation out.